modules/nuxt-viewport.yml

```diff
@@ -0,0 +1,14 @@
+name: nuxt-viewport
+description: Define custom viewports for your Nuxt project
+repo: mvrlin/nuxt-viewport
+npm: nuxt-viewport
+icon: nuxt-viewport.svg
+github: https://github.com/mvrlin/nuxt-viewport
+website: https://github.com/mvrlin/nuxt-viewport
+learn_more: ''
+category: Libraries
```

Comment:
Library is for modules specifically integrate a js/vue library. We might use extensions or ui for this module

Reply:
@pi0 Done :)